main.py

```diff
@@ -269,7 +273,6 @@ def load_settings():
     show_view_status = settings_obj.get("show_view_status", True)
     auto_show_diagnostics_panel = settings_obj.get("auto_show_diagnostics_panel", True)
     show_diagnostics_phantoms = settings_obj.get("show_diagnostics_phantoms", True)
```

Comment:
Yes, it's duplication, but please change this to match the settings default.
We can get rid of the defaults in `settings_obj.get('my_key')` in a future PR.